### PR TITLE
Close secrets and keys files after the content has been read

### PR DESCRIPTION
**Pull request by jonnyshaw89**
### What is the context of this PR?
The Keys and Secrets file resources were not being closed after the file was read.
This PR ensures that they are closed

### CODE REVIEW

File: app/setup.py [hunk 1]
```diff
@@ -64,8 +64,12 @@ def create_app(setting_overrides=None):  # noqa: C901  pylint: disable=too-compl
 
     application.eq = {}
 
-    secrets = yaml.safe_load(open(application.config['EQ_SECRETS_FILE']))
-    keys = yaml.safe_load(open(application.config['EQ_KEYS_FILE']))
+    with open(application.config['EQ_SECRETS_FILE']) as secrets_file:
+        secrets = yaml.safe_load(secrets_file)
+
+    with open(application.config['EQ_KEYS_FILE']) as keys_file:
+        keys = yaml.safe_load(keys_file)
```
Comment (shahi645) [inline app/setup.py:71]: @jonnyshaw89 This may be a silly question but how does this ensure they get closed ?

Comment (jonnyshaw89) [reply app/setup.py:71]: Have a google

Comment (shahi645) [reply app/setup.py:71]: 🥇

**Review by shahi645 — APPROVED** [review]

File: app/setup.py [hunk 2]
```diff
@@ -64,8 +64,12 @@ def create_app(setting_overrides=None):  # noqa: C901  pylint: disable=too-compl
 
     application.eq = {}
 
-    secrets = yaml.safe_load(open(application.config['EQ_SECRETS_FILE']))
-    keys = yaml.safe_load(open(application.config['EQ_KEYS_FILE']))
+    with open(application.config['EQ_SECRETS_FILE']) as secrets_file:
+        secrets = yaml.safe_load(secrets_file)
+
```
Comment (MIslam92) [inline app/setup.py:69]: Should you not be calling the close method the file like this i.e. secrets.close()? as best practice or does it auto close?

**Review by MIslam92 — CHANGES_REQUESTED** [review]

File: app/setup.py (hunk 2 above)
Comment (jonnyshaw89) [reply app/setup.py:69]: http://effbot.org/zone/python-with-statement.htm

Comment (MIslam92) [reply app/setup.py:69]: ahhh i was looking at: http://www.tutorialspoint.com/python/python_files_io.htm but forgot bout the with part

**Review by MIslam92 — APPROVED** [review]

**Pull request merged**